src/main/java/org/opensrp/web/controller/MultimediaController.java

```diff
@@ -259,4 +288,22 @@ private void writeFileNotFound(HttpServletResponse response) throws IOException
 		outputStream.write(errorMessage.getBytes(Charset.forName("UTF-8")));
 		outputStream.close();
 	}
+
+	private void fileWithSpecialCharactersError(HttpServletResponse response) throws IOException {
+		String errorMessage = "Sorry. File Name should not contain any special character";
+		logger.error(errorMessage);
+		OutputStream outputStream = response.getOutputStream();
+		response.setStatus(HttpStatus.BAD_REQUEST.value());
+		outputStream.write(errorMessage.getBytes(Charset.forName("UTF-8")));
+		outputStream.close();
+	}
+
+	private void entityIdWithSpecialCharactersError(HttpServletResponse response) throws IOException {
+		String errorMessage = "Sorry. Entity Id should not contain any special character";
+		logger.error(errorMessage);
+		OutputStream outputStream = response.getOutputStream();
+		response.setStatus(HttpStatus.BAD_REQUEST.value());
+		outputStream.write(errorMessage.getBytes(Charset.forName("UTF-8")));
+		outputStream.close();
+	}
```

Comment:
This method is similar to the one above, combine them and pass errorMessage as a Param

Reply:
Updated.